mobile/node.go

```
@@ -108,6 +101,23 @@ func NewNode(config NodeConfig) (*Node, error) {
 		return nil, err
 	}
 
+	ct := wallet.Bitcoin
+	switch walletCfg.Type {
+	case "bitcoincash":
+		ct = wallet.BitcoinCash
+	case "zcashd":
+		ct = wallet.Zcash
+	}
+	migrations.WalletCoinType = ct
+
+	sqliteDB, err := initializeRepo(config.RepoPath, "", "", true, time.Now(), ct)
+	if err != nil && err != repo.ErrRepoExists {
+		return nil, err
+	}
+
+	// Get creation date. Ignore the error and use a default timestamp.
+	creationDate, _ := sqliteDB.Config().GetCreationDate()
```

Don't ignore the error, please.

---

See above